### ISSUE TITLE
ChinaVine video not working

### ISSUE THREAD

**Issue by jsimic** [OregonDigital/oregondigital#833]
@terrellt 
http://oregondigital.library.oregonstate.edu/catalog/oregondigital:fx71b483g

**Comment by terrellt** [comment]
Missing a codec. Recompiling ffmpeg.

**Comment by terrellt** [comment]
Alright, looks like this worked this time. Try uploading it again and make sure it can do it automatically?

**Comment by jsimic** [comment]
@terrellt  Does not seem to have worked. Uploaded it about 2.5 hours ago.

http://oregondigital.library.oregonstate.edu/catalog/oregondigital:fx71b4851

**Comment by terrellt** [comment]
Other derivatives are holding up the queue

**Comment by lsat12357** [comment]
I think it's still busted.

**Comment by terrellt** [comment]
I think I have this fixed. Requeueing derivatives for ChinaVine

**Comment by sseymore** [comment]
@terrellt The uploaded videos have no audio but we discovered that the local batch from .mov to .m4v was corrupted. Julia fixed it and I manually updated these items but still no audio. Thoughts? Does it take a while to resolve? Let me know if this is confusing. 

http://oregondigital.library.oregonstate.edu/catalog/oregondigital:df67tw73x

http://oregondigital.library.oregonstate.edu/catalog/oregondigital:df67tw62d
